wp-parsely.php

```diff
@@ -34,20 +39,41 @@ function wpvip_load_wp_parsely_plugin() {
 		return;
 	}
 
+	$versions_to_try = SUPPORTED_VERSIONS;
+
 	/**
 	 * Allows specifying a major version of the plugin per-site.
 	 * If the version is invalid, the default version will be used.
 	 */
-	$major_version = apply_filters( 'wpvip_parsely_major_version', WPVIP_PARSELY_DEFAULT_VERSION );
-	if ( ! in_array( $major_version, [
-		'2.5',
-	] ) ) {
+	$specified_version = apply_filters( 'wpvip_parsely_version', false );
+
+	if ( in_array( $specified_version, SUPPORTED_VERSIONS ) ) {
+		array_unshift( $versions_to_try, $specified_version );
+		$versions_to_try = array_unique( $versions_to_try );
+	} else {
 		trigger_error(
-			sprintf( 'Invalid value configured via wpvip_parsely_major_version filter: %s', $major_version ),
+			sprintf( 'Invalid value configured via wpvip_parsely_version filter: %s', $version ),
```

Comment:
Not sure why this is hitting `WordPress.Security.EscapeOutput.OutputNotEscaped` in phpcs -- I'm not outputting anything 🤔 

https://app.circleci.com/pipelines/github/Automattic/vip-go-mu-plugins/6713/workflows/3faa430e-6757-45d4-ab8d-6d847fc0d4a6/jobs/46847

Reply:
You are as part of the `trigger_error()`.

See https://github.com/WordPress/WordPress-Coding-Standards/blob/dc2f21771cb2b5336a7e6bb6616abcdfa691d7de/WordPress/Sniffs/Security/EscapeOutputSniff.php#L198